ledger/participant-integration-api/src/main/scala/platform/index/BuffersUpdater.scala

```diff
@@ -48,7 +47,7 @@ private[index] class BuffersUpdater(
   private val logger = ContextualizedLogger.get(getClass)
 
   private[index] val updaterIndex = new AtomicReference(
-    Offset.beforeBegin -> EventSequentialId.beforeBegin
+    Offset.beforeBegin -> 0L // TODO: append-only: FIXME consolidating parameters table
```

Comment:
Why are you inlining this constant?

Reply:
The original object is deleted, so I just inlined temporarily with a TODO to tackle this upon parameters table consolidation.